context/app/routes_cells.py

```diff
@@ -94,7 +94,7 @@ def _get_cluster_name_and_number(cluster_str):
     return Cluster(name=cluster_name, number=cluster_number)
 
 
-def _get_cluster_cells(cells, gene, min_gene_expression):
+def _get_cluster_cells(cells, query_type, name, min_expression):
```

Comment:
Not a hard cut-off, but somewhere between 3-5 args we should start using some kwargs.